pkgs/games/julius/default.nix

```diff
@@ -5,8 +5,15 @@
 , SDL2_mixer
 , cmake
 , libpng
+, runCommand
+, darwin ? null
 }:
-
+let
+  darwinSymlinks = runCommand "julius-darwin-build-symlinks" { } ''
+    mkdir -p $out/bin
+    ln -s /usr/bin/sips /usr/bin/iconutil /usr/bin/codesign $out/bin
```

Comment:
Wouldn't it be easier to append /usr/bin to PATH?

Reply:
Got inspired by a similar approach used in`macvim`: https://github.com/NixOS/nixpkgs/blob/nixos-24.05/pkgs/applications/editors/vim/macvim.nix#L18-L27

Happy to add it to the PATH if it's a better solution